runtime/jvmti/j9jvmti.tdf

```diff
@@ -639,3 +638,5 @@ TraceExit=Trc_JVMTI_jvmtiHookVirtualThreadStarted_Exit Overhead=1 Level=5 Noenv
 
 TraceEntry=Trc_JVMTI_jvmtiHookVirtualThreadDestroy_Entry Overhead=1 Level=5 Noenv Template="HookVirtualThreadDestroy"
 TraceExit=Trc_JVMTI_jvmtiHookVirtualThreadDestroy_Exit Overhead=1 Level=5 Noenv Template="HookVirtualThreadDestroy"
+
+TraceAssert=Assert_JVMTI_unreachable noEnv Overhead=1 Level=1 Assert="0"
```

Comment:
`Assert_JVMTI_true` is already being used for the unreachable case below:
https://github.com/eclipse-openj9/openj9/blob/ae516282886b96f3015cb2ea03f5bf8204c93f4f/runtime/jvmti/jvmtiClass.c#L2688-L2691

Since we plan to keep the new tracepoint, we should also use the new tracepoint in the above case for consistency.

Reply:
Yes, consistency would be good.

Reply:
We can do the rest in a subsequent PR.